vcr/init/init.go

```diff
@@ -23,14 +23,16 @@ import (
 )
 
 const SkipValue = "SKIP"
+const DefaultRuntime = "nodejs18"
```

Comment:
are these used outside the package? if not we should make sure they are not capitalised

Reply:
this value will be only used in init command since it will give options for users to choose which runtime they want to use in vcr.yaml. this value is to match the one in graphql like below
   {
        "language": "nodejs",
        "name": "nodejs18",
        "comments": null
      },